File: chap07/run-model7-5.R
library(cmdstanr)

d <- read.csv(file='input/data-50m.csv')
data <- list(N=nrow(d), Age=d$Age, Weight=d$Weight, Y=d$Y)

model <- cmdstan_model(stan_file='model/model7-5.stan')
fit <- model$sample(data=data, seed=123, parallel_chains=4)
fit$save_object(file='output/result-model7-5.RDS')

File: chap07/run-model7-5.py
import pandas
import cmdstanpy

d = pandas.read_csv('input/data-50m.csv')
data = {'N':len(d), 'Age':d.Age, 'Weight':d.Weight, 'Y':d.Y}

model = cmdstanpy.CmdStanModel(stan_file='model/model7-5.stan')
fit = model.sample(data=data, seed=123, parallel_chains=4)
fit.save_csvfiles('output/result-model7-5')
